media-gfx/openvdb/openvdb-7.1.0-r2.ebuild

```diff
@@ -5,7 +5,7 @@ EAPI=7
 
```

Comment:
We should revbump for this change as it fixes how/which tools get built.

Reply:
@thesamesam thanks for revbumping. I was on a vacation today, so couldn't react that fast.